tests/features/mocking_async.py

```diff
@@ -0,0 +1,201 @@
+"""Tests for flexmock mocking feature."""
+# pylint: disable=missing-docstring,no-self-use,no-member,too-many-lines
+import unittest
+import pytest
+
+from flexmock import exceptions, flexmock
+from tests import some_module
+from tests.utils import assert_raises
```

Comment:
Looks like this is not properly formatted, I think it's simply
```suggestion
import unittest

import pytest

from flexmock import exceptions, flexmock
from tests import some_module
from tests.utils import assert_raises
```
but you can run isort yourself to check